### PR TITLE
Add example newsfeed to the sample app

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

File: BridgeAppSDKSample/Info.plist [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>blogspot.com</key>
+			<dict>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+				<key>NSThirdPartyExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+			<key>sagebase.org</key>
```
Comment (Erin-Mounts) [inline BridgeAppSDKSample/Info.plist]: does the sample app actually access any urls at sagebase.org?

File: BridgeAppSDKSample/Info.plist [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+			<dict>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+				<key>NSThirdPartyExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+			<key>pdf.org</key>
```
Comment (Erin-Mounts) [inline BridgeAppSDKSample/Info.plist]: does the sample app actually access any urls at pdf.org (Parkinson's Disease Foundation)?

File: BridgeAppSDKSample/NewsfeedTableViewController.swift [hunk 3]
```diff
@@ -0,0 +1,116 @@
+//
+//  Smart4SURENewsfeedTableViewController.swift
+//  Smart4SURE
```
Comment (Erin-Mounts) [inline BridgeAppSDKSample/NewsfeedTableViewController.swift]: Smart4SURE --> BridgeAppSDK

**Review by Erin-Mounts — COMMENTED** [review]
Questions about URL security exceptions, header comments copypasted from Smart4SURE without updating references to file & project name

File: BridgeAppSDKSample/Info.plist (hunk 2 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDKSample/Info.plist]: I think that was added to Parkinson for PDF support.

File: BridgeAppSDKSample/Info.plist (hunk 1 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDKSample/Info.plist]: I don't think so.

File: BridgeAppSDKSample/NewsfeedTableViewController.swift (hunk 3 above)
Comment (syoung-smallwisdom) [reply BridgeAppSDKSample/NewsfeedTableViewController.swift]: whoops.

File: BridgeAppSDKSample/Info.plist (hunk 2 above)
Comment (Erin-Mounts) [reply BridgeAppSDKSample/Info.plist]: That URL is for the Parkinson's Disease Foundation (PDF)... I know, it's confusing :)

**Comment by Erin-Mounts** [comment]
looks good

**Pull request merged**